lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -10,8 +10,8 @@
 import subprocess as sp
 
 output_dir = os.path.join(test_dir, 'scripts')
-dl1_file = os.path.join(output_dir, 'dl1_gamma_test_large.simtel.h5')
```

Comment:
Just for people to know, we are also getting rid of the `simtel` string:
https://github.com/cta-observatory/cta-lstchain/pull/344#discussion_r410127986

Reply:
I can put it back in, if you want. There was some previous discussions on it. The behaviour before was not well-defined. It would be removed for uncompressed simtel files (without .gz or .zst) but left in for compressed simtel files.

Reply:
no, I'm fine with this, as I said in the comment from the previous PR, but just wanted for people dealing with simulations to be aware